setup.py

```diff
@@ -1,29 +1,13 @@
 from setuptools import setup
 import os
-import subprocess as sub
+from freecad.plot.PlotGui import CompileResources
```

Comment:
I am not sure if this import works if freecad.plot is not installed. I guess something similar to line 5 in this file is needed.

Reply:
It seems to work for me. If finds the import even if run from outside FreeCAD. It is simply importing from the plot subdirectory of the freecad subdirectory rather than from a freecad.plot package. But I could well be missing a sublety.

Reply:
You are right. This is not a problem.
thanks for the work!